### ISSUE TITLE
[FEATURE]: Add 'Neural Network' as model creation option.

### ISSUE THREAD

**Issue by alantao912** [DSGT-DLP/Deep-Learning-Playground#1108]
### Feature Name

Create Feedforward Neural Networks

### Your Name

Alan Tao

### Description

Allow users to create feed forward neural networks by clicking on 'Create Model' on the trainspace dashboard.

Clicking on the 'Create Model' button, should lead the user to a configuration screen where they should be allowed to interactively drag and drop layers to the neural network as well as modify the details of each layer including number of neurons and activation function.

The same configuration screen should also prompt the user for all of the existing parameters shown in this screenshot:
![image](https://github.com/DSGT-DLP/Deep-Learning-Playground/assets/70001632/c6e25157-cf96-47cf-9321-8982a790e108)

There should be a visualization of the network's architecture that is updated in real time as the user modifies parameters/adds more layers. Something that looks similar to this:

![Untitled](https://github.com/DSGT-DLP/Deep-Learning-Playground/assets/70001632/e10b0eff-327d-4f3e-b60d-64da2b216248)

In other words, all of the existing configuration steps when creating the current TABULAR model (excluding the dataset upload step), should be combined into a single configuration page. This is an open ended task so feel free to apply your own creativity toward designing the layout of this monolithic configuration page. PR reviewers will iteratively provide feedback.

**Comment by alantao912** [comment]
Whoever the assignee is, feel free to reach out to me for clarification.

**Comment by karkir0003** [comment]
@alantao912, how's this task coming along? what's the priority of this one?
